pipelines.yml

```diff
@@ -169,4 +168,15 @@ pipelines:
           onExecute:
             - cd $res_jfrogCliGit_resourcePath
             - *CREATE_RT_ACCESS_TOKEN
-            - env -i PATH=$PATH M2_HOME=$M2_HOME HOME=$HOME `which go` test -v -timeout 0 -test.plugins -rt.url=$int_jfrog_rt_url -rt.accessToken=$(< rtToken)
+            - env -i PATH=$PATH M2_HOME=$M2_HOME HOME=$HOME `which go` test -v -timeout 0 -test.plugins -jfrog.url=$int_jfrog_rt_url -jfrog.accessToken=$(< rtToken)
+
+      - name: Xray
+        type: Bash
+        configuration:
+          <<: *COMMON_CONFIGURATION
+        execution:
+          <<: *UPDATE_COMMIT_STATUS
+          onExecute:
+            - cd $res_jfrogCliGit_resourcePath
+            - *CREATE_RT_ACCESS_TOKEN
+            - env -i PATH=$PATH M2_HOME=$M2_HOME HOME=$HOME `which go` test -v -timeout 0 -test.xray -jfrog.url=$int_jfrog_rt_url -jfrog.accessToken=$(< rtToken)
```

Comment:
This will probably not work. The Access token created is valid only for the Artifactory service.